code/game/objects/items/implants/implantuplink.dm

```diff
@@ -21,3 +31,6 @@
 
 /obj/item/implant/uplink/precharged
 	starting_tc = 10
+
+/obj/item/implant/uplink/starting
+	starting_tc = 16  /// default TC count 20, minus cost of implant (4)

```

Comment:
there is a potential issue here where the uplink implant cost gets changed and they forget to adjust this.

Reply:
Yeah, should be defines somewhere for default starting TC and the cost of the implant. If not, create them please